src/visitpy/common/visitmodule.C

```diff
@@ -1305,6 +1309,9 @@ CreateDictionaryFromDBOptions(DBOptionsAttributes &opts)
           case DBOptionsAttributes::String:
             PyDict_SetItemString(dict,name,PyString_FromString(opts.GetString(name).c_str()));
             break;
+          case DBOptionsAttributes::MultiLineString:
+            PyDict_SetItemString(dict,name,PyString_FromString(opts.GetMultiLineString(name).c_str()));
+            break;
```

Comment:
Do you think adding a `default` is at all useful like I tried here...

https://github.com/visit-dav/visit/blob/a5d504da469a4e181a5aa832ae8ec3aae59c03dc/src/visitpy/common/visitmodule.C#L1331-L1338

Reply:
It might be good to add, to help us catch the case in the future where a new type is added but not handled. I can try it out if you like. Should your approval be dismissed in the meantime?

Reply:
> Should your approval be dismissed in the meantime?

No.